roles/confluent.preflight/tasks/debian.yml

```diff
@@ -1,12 +1,31 @@
-- name: Add open JDK repo
-  apt_repository:
-    repo: ppa:openjdk-r/ppa
+- name: Copy in repo skip valid check
+  copy:
+    content: 'Acquire::Check-Valid-Until "0";'
+    dest: /etc/apt/apt.conf.d/skip-check
+
+- name: Add jessie-backports Repo
+  lineinfile:
+    path: /etc/apt/sources.list
+    line: deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main
+    regexp: '.*jessie-backports.*'
+  notify:
+    - debian apt-get update
+
+- meta: flush_handlers
+
 - name: Install Java
   apt:
-    name: "openjdk-8-jdk"
-    update_cache: yes
-- name: Disable Require TTY in Sudoers (required for ansible synchronize)
-  lineinfile:
-    path: /etc/sudoers
-    state: absent
-    regexp: '^Defaults\s+requiretty.*$'
+    name: openjdk-8-jdk
+    state: present
+    default_release: jessie-backports
+
+- name: Install rsync
```

Comment:
rename?

Reply:
Can you clarify what needs to be renamed here?